src/Compilers/CSharp/Test/Emit/CodeGen/PatternTests.cs

```diff
@@ -5496,7 +5496,7 @@ class B
 
         #region Pattern Combinators
 
-        [Fact]
+        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/55334")]
```

Comment:
I'd prefer to not skip these. If the il regresses as part of this change, we should capture that information. The skip implies this test is actually broken.

Reply:
I would prefer if we avoided ever skipping tests, unless the behavior is so bad that it does something like time out or crash the test process, for example.

Reply:
Done. Please take a look, thanks.